### PR TITLE
Prepare for WRF-v4.1.1 release

### PR DESCRIPTION
**Pull request by smileMchen**
TYPE: text only

KEYWORDS: README, release

SOURCE: internal

DESCRIPTION OF CHANGES: 
Update the top-level README file, and the inc/version_decl file with the correct version.
Now using a string which in no longer a fixed length.

LIST OF MODIFIED FILES: 
M	   README
M	   inc/version_decl

TESTS CONDUCTED:  A test case is run with the correct printout
```
taskid: 0 hostname: r5i2n23
Quilting with   1 groups of   0 I/O tasks.
 Ntasks in X           12 , ntasks in Y           12
WRF V4.1.1 MODEL
 *************************************
 Parent domain
 ids,ide,jds,jde            1         401           1         261
 ims,ime,jms,jme           -4          41          -4          29
 ips,ipe,jps,jpe            1          34           1          22
 *************************************
```
 - [x] Do a run and demonstrate that the printout is correct.

### CODE REVIEW

File: inc/version_decl [hunk 1]
```diff
@@ -1 +1 @@
-   CHARACTER (LEN=10) :: release_version = 'V4.1      '
+   CHARACTER (LEN=10) :: release_version = 'V4.1.1      '
```
Comment (mgduda) [inline inc/version_decl]: Here we're assigning a string of length 12 to a variable of declared length 10.

Comment (mgduda) [reply inc/version_decl]: I think it's valid Fortran to write something like this:
```fortran
CHARACTER (LEN=*), PARAMETER :: release_version = 'V4.1.1' 
```
This way, we don't need to worry about putting in the right number of spaces. However, I don't know whether there is downstream code that depends on `release_version` being exactly 10 characters or that needs to change `release_version`.

Comment (smileMchen) [reply inc/version_decl]: @mgduda 
Michael,
Thanks a lot. i have changed the code as suggested.

File: inc/version_decl [hunk 2]
```diff
@@ -1 +1 @@
-   CHARACTER (LEN=10) :: release_version = 'V4.1      '
+   CHARACTER (LEN=10) :: release_version = 'V4.1.1'
```
Comment (davegill) [inline inc/version_decl]: Ming,
Michael suggested to also change the `LEN=10` to `LEN=*`.

Would you make this change, and then run the WRF model and look at the print out. We should see something like:
```
WRF V4.1.1 MODEL
```
Update the commit message that you have conducted this test.

Comment (davegill) [reply inc/version_decl]: @jamiebresch 
... and with a parameter statement, too

**Comment by davegill** [comment]
@smileMchen 
Ming,
Do a run and demonstrate that the printout is correct.
Dave

**Comment by smileMchen** [comment]
@davegill 
I run a test case and the printout is correct. Please approve this PR.
Ming

**Comment by davegill** [comment]
@kkeene44 @smileMchen 
Kelly,
I _want_ to approve this, but the global owner is required.
I asked for a test, and Ming conducted it. Ming has modified the code to reflect Michael's suggestions.

**Comment by smileMchen** [comment]
TYPE: text only

KEYWORDS: README, release

SOURCE: internal

DESCRIPTION OF CHANGES:
Update the top-level README file, and the inc/version_decl file with the correct version.
Now using a string which in no longer a fixed length.

LIST OF MODIFIED FILES:
M	README
M	inc/version_decl

TESTS CONDUCTED: A test case is run with the correct printout

taskid: 0 hostname: r5i2n23
Quilting with   1 groups of   0 I/O tasks.
 Ntasks in X           12 , ntasks in Y           12
WRF V4.1.1 MODEL
 *************************************
 Parent domain
 ids,ide,jds,jde            1         401           1         261
 ims,ime,jms,jme           -4          41          -4          29
 ips,ipe,jps,jpe            1          34           1          22
 *************************************

**Pull request merged**